### DISCUSSION TITLE
Why can I toggle the assistant role to "Assistant" or "System"?

### DISCUSSION THREAD

**Discussion by xpe** [zed-industries/zed#12465] [Feature Requests]
This part of the chatbot assistant feature confuses me.

- I type questions under the "You" role. (Meaning me, the user)
- I understand why responses come from the "Assistant" role.
- I've never seen anything appear from "System".
- Why does Zed let me toggle between the three? I don't follow why anyone would do this.

**Comment by xpe** [comment]
Ah, from https://zed.dev/docs/assistant-panel

> You can cycle the role of a message block by clicking on the role, which is useful when you receive a response in an Assistant block that you want to edit and send back up as a You block.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
